api/endpoint-tests/endpoint.sh

```diff
@@ -6,7 +6,7 @@ rm ./endpoint-tests/endpoints-hit.txt
 docker-compose -f docker-compose.endpoint-tests.yml up -d
 docker-compose -f docker-compose.endpoint-tests.yml exec api-for-testing npm run migrate
 docker-compose -f docker-compose.endpoint-tests.yml exec api-for-testing npm run seed
-docker-compose -f docker-compose.endpoint-tests.yml exec api-for-testing npm run test-endpoints
+docker-compose -f docker-compose.endpoint-tests.yml exec api-for-testing npm run test-endpoints $@
```

Comment:
This passes along arguments from the command line into the actual test.  That way you can update snapshots using the helper script:

```shell
./endpoint-tests/endpoint.sh -- -u
```